tower-balance/src/p2c/service.rs

```diff
@@ -137,18 +151,20 @@ where
         }
     }
 
-    fn poll_unready(&mut self) {
+    fn poll_unready(&mut self, cx: &mut Context<'_>) {
```

Comment:
Pretty sure this'll need to take `Pin<&mut Self>`

Reply:
Oh, I guess since `Service` is all `&mut self` for the time being, it doesn't?

Reply:
Yup, and it seems to work out pretty well!